Source/WebCore/page/Page.cpp

```diff
@@ -1694,7 +1696,7 @@ void Page::computeUnfulfilledRenderingSteps(OptionSet<RenderingUpdateStep> reque
 void Page::triggerRenderingUpdateForTesting()
 {
     LOG_WITH_STREAM(EventLoop, stream << "Page " << this << " triggerRenderingUpdateForTesting()");
-    renderingUpdateScheduler().triggerRenderingUpdateForTesting();
```

Comment:
This patch weakens the role of RenderingUpdateScheduler. Can we move the calls to ChromeClient to inside that class?

Reply:
As discussed offline, RenderingUpdateScheduler is a DisplayRefreshMonitorClient, which isn't supported as of this change (the relevant monitor implementation being removed).

This change gives ChromeClient the first opportunity to handle rendering update requests, and falls back to RenderingUpdateScheduler.